<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.3.2 (Python 3.13.6 on win32)" generated="2025-08-12T00:16:05.859907" rpa="false" schemaversion="5">
<suite id="s1" name="Common" source="C:\python_study\OTT\Robot_frework_OTT\resource\keywords\common.robot">
<test id="s1-t1" name="Connect to Emulator" line="5">
<kw name="Connect Device">
<msg time="2025-08-12T00:16:06.593081" level="FAIL">No keyword with name 'Connect Device' found.</msg>
<arg>emulator</arg>
<status status="FAIL" start="2025-08-12T00:16:06.593038" elapsed="0.000099">No keyword with name 'Connect Device' found.</status>
</kw>
<tag>emulator</tag>
<status status="FAIL" start="2025-08-12T00:16:06.591342" elapsed="0.002031">No keyword with name 'Connect Device' found.</status>
</test>
<status status="FAIL" start="2025-08-12T00:16:05.864416" elapsed="0.730408"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
<stat pass="0" fail="1" skip="0">emulator</stat>
</tag>
<suite>
<stat name="Common" id="s1" pass="0" fail="1" skip="0">Common</stat>
</suite>
</statistics>
<errors>
<msg time="2025-08-12T00:16:06.589685" level="WARN">Imported library 'C:\python_study\OTT\Robot_frework_OTT\resource\keywords\Setup_utility.py' contains no keywords.</msg>
</errors>
</robot>
